loggo/loggo.py

```diff
@@ -34,7 +34,24 @@ class Loggo:
     # Only log when log level is this or higher
     log_threshold = logging.DEBUG
 
-    def __init__(self, config: Optional[Dict] = None) -> None:
+    def __init__(self,
+                 called: Optional[str] = FORMS['called'],
+                 returned: Optional[str] = FORMS['returned'],
+                 returned_none: Optional[str] = FORMS['returned_none'],
+                 errored: Optional[str] = FORMS['errored'],
+                 error_level: int = 20,
+                 facility: str = 'loggo',
+                 ip: Optional[str] = None,
+                 port: Optional[str] = None,
```

Comment:
This used to be and should be an `Optional[int]`

Reply:
might also want to consider renaming/retyping to something like `graylog_address: Optional[Tuple[str, int]] = None`.

Reply:
First comment, sure, second comment, make new issue

Reply:
note: error_level is now optional, but i had to add `level = self.error_level if self.error_level is not None else logging.INFO` so that the None actually does something...

Reply:
Why allow `None` for error_level btw? We could have it be strictly an int, and with the default value it can still be optional.

Reply:
Also, port seems to still be of type string in the PR.